main_test.go

```diff
@@ -2,19 +2,67 @@ package main
 
 import (
 	"testing"
+
+	"gorm.io/gorm"
 )
 
 // GORM_REPO: https://github.com/go-gorm/gorm.git
 // GORM_BRANCH: master
 // TEST_DRIVERS: sqlite, mysql, postgres, sqlserver
 
 func TestGORM(t *testing.T) {
-	user := User{Name: "jinzhu"}
+	type Hoge struct {
+		ID   uint
+		Name string
+	}
+	type Fuga struct {
+		ID   uint
+		Name string
+	}
 
-	DB.Create(&user)
+	if err := DB.AutoMigrate(&Hoge{}, &Fuga{}); err != nil {
+		t.Errorf("%v", err)
+	}
 
-	var result User
-	if err := DB.First(&result, user.ID).Error; err != nil {
-		t.Errorf("Failed, got error: %v", err)
+	DB = DB.Session(&gorm.Session{})
+	DB = DB.InstanceSet("key", "value") // Correct without this line
+	// DB = DB.Debug() // Correct with this line
+	if err := DB.Create(&Hoge{Name: "Alice"}).Error; err != nil {
```

Comment:
I'm sorry. I couldn't write the test well to fail.
Please see the Debug output.